src/main/java/org/broadinstitute/hellbender/tools/picard/analysis/AlignmentSummaryMetricsCollector.java

```diff
@@ -124,6 +126,8 @@ private boolean isAdapterSequence(final byte[] read) {
     }
 
     private class GroupAlignmentSummaryMetricsPerUnitMetricCollector implements PerUnitMetricCollector<AlignmentSummaryMetrics, Long, SAMRecordAndReference> {
```

Comment:
Not without making a bunch of other changes. All of these (other Picard) metrics classes will all have to be ported to hellbender style and the new metrics framework, but those will be separate PRs.